src/screens/Shared/EditLocation.js

```diff
@@ -168,7 +172,7 @@ class EditLocation extends Component {
                 >
                   <Content>
                     <Text>{result.address_1}</Text>
-                    <Text>{result.city}, {result.state} {result.zipcode}</Text>
+                    <Text>{this.calloutBody(result)}</Text>
```

Comment:
I like that this is factored out now, but if the callout ends up being more complicated/more of a component, it might end up having to be declared as a var in the return.